### DISCUSSION TITLE
Group columns?

### DISCUSSION THREAD

**Discussion by OvalMedia** [filamentphp/filament#5500] [Help]
Is it possible to group multiple table columns into one?
In my case I have columns that are only represented by a symbol (BooleanColumn) but by each having their own table column they are using up more space than needed.

It would be nice if those could be rendered inside a single html column.

**Comment by Samoylenko666** [comment]
alternatively i think you can use accessor in your model https://laravel.com/docs/master/eloquent-mutators#defining-an-accessor

**Comment by lukevi** [comment]
the first answer is probably the best way. another way is to use ->state() to combine them, like so:

```php
                Tables\Columns\TextColumn::make('options')
                    ->state(fn ($record) => ($record->bool1 ? 'Y' : 'N') . ' ' . ($record->bool2 ? 'Y' : 'N')),
```

**Comment by getupkid** [comment]
Perhaps try a Custom Column?
https://filamentphp.com/docs/3.x/tables/columns/custom
